components/icons/ArrowIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from "react"
+
+interface Props {
+    onClick?: () => void
+    className?: string
+}
+
+export const ArrowIcon:FC<Props> = ({ onClick, className }) => {
+    return (
+        <svg
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke-width="1.5"
+            stroke="currentColor"
+            className={"w-6 h-6 cursor-pointer" + (className ? ` ${className}` : '')}
```

Comment:
ah yes, this works but you're gonna find some issues with this direct approach of string templates with tailwind classes. I typically use a combination of the npm package `classnames` to better handle conditional classes as well as the package `tailwind-merge` because the order of classnames matters when you have two of them that specify different values for the same css property.

What I do with those two libs is this util function:

```
import { twMerge } from "tailwind-merge";
import classnames from "classnames";

export const tw: typeof classnames = (...params) =>
  twMerge(classnames(...params));

```

It works exactly like classnames` api but it also enforces the same order of tailwind classes so you don't have inconsistent results